src/main/java/teammates/ui/output/FeedbackSessionLogEntry.java

```diff
@@ -0,0 +1,33 @@
+package teammates.ui.output;
+
+/**
+ * The session log of a student for a single feedback session.
+ */
+public class FeedbackSessionLogEntry {
+    private final StudentData studentData;
+
+    private final String feedbackSessionLogType;
+
+    private final long timestamp;
+
+    public FeedbackSessionLogEntry(StudentData studentData,
+                                   String feedbackSessionLogType,
+                                   long timestamp) {
+        this.studentData = studentData;
+        this.feedbackSessionLogType = feedbackSessionLogType;
+        this.timestamp = timestamp;
+    }
+
+    public StudentData getStudentData() {
+        return studentData;
+    }
+
+    public String getFeedbackSessionLogType() {
+        // TODO: change type from String to LogType
```

Comment:
Any particular reason is this marked as a TODO? What preventing the use of `LogType`?